sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/ReleaseInstallerListenerTest.java

```diff
@@ -331,4 +331,22 @@ public void releaseInstallerProcessWhenWithContext() throws Exception {
         /* Verify that installer process was triggered in the Distribute again. */
         verify(mDistribute).notifyInstallProgress(eq(false));
     }
+
+    @Test
+    public void releaseInstallerHideDialogTwice() {
+        /* Start install process. */
+        mReleaseInstallerListener.startInstall();
+
+        /* Hide dialog. */
+        mReleaseInstallerListener.hideInstallProgressDialog();
+
+        /* Try to hide dialog again */
+        mReleaseInstallerListener.hideInstallProgressDialog();
+
+        /* Verify that runnable was called once only. */
+        ArgumentCaptor<Runnable> runnable = ArgumentCaptor.forClass(Runnable.class);
+        verifyStatic(times(1));
+        HandlerUtils.runOnUiThread(runnable.capture());
+        runnable.getValue().run();
```

Comment:
Do we really need these lines? What is it checking?

Reply:
This checks that the code of hiding dialog is invoking only once. `HandlerUtils.runOnUiThread(new Runnable() {` in `hideInstallProgressDialog`.